docs/making/regions.md

```diff
@@ -1 +1,85 @@
-Coming soon.
+# Regions
+
+Regions are a powerful and simple way to organize your game logic and make it easier to understand and edit. In essence, regions allow you to take large groups of locations that have something in common and make them share the same logic and requires.
+
+```json
+[
+    "Overworld": {
+        "starting": true,
+        "connects_to": ["Lonely Island","Spooky Manor"],
+        "requires": []
+    },
+    "Lonely Island": {
+        "connects_to": [],
+        "requires": "|Ferry Ticket|"
+    },
+    "Spooky Manor": {
+        "connects_to": ["Manor Basement"],
+        "requires": "|Manor Key| OR |Wall Climb|"
+    },
+    "Manor Basement": {
+        "connects_to": [],
+        "requires": "|Basement Key| AND |Lantern|"
+    }
+]
+```
+
+## The basics
+
+When you create a [location](./locations.md), you can optionally assign it to a region. When you do this, the location will inherity ALL logic from the region it is in. This means that access to that location is not _only_ determined by that location's requires, but it now _also_ obeys the requires of the region it is in, _as well as_ that region's connection requirements (see below). If you have a location that is set to require Item A, and that location is in region "Cool Region", and "Cool Region" is set to require Item B, __that location will now require both Item A AND Item B before it is considered accessible__.
+
+_In locations.json:_
+
+```json
+    {
+        "name": "My Location",
+        "region": "Cool Region",
+        "requires": "Item A"
+    },
+```
+
+_In regions.json:_
+
+```json
+    "Cool Region": {
+        "requires": ["Item B"]

```

Comment:
We should use the boolean req pipes for both of these, since we want to deprecate legacy reqs at some point. (Also, the first req doesn't work as-is.)